### PR TITLE
python3Packages.skybellpy: init at 0.6.3

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python wrapper for the Skybell alarm API

https://github.com/MisterWil/skybellpy

This is a Home Assistant dependency.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112991` at 6d78f158 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>python39Packages.skybellpy</li>
  </ul>
</details>
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/d12141032db5864085af39ce1df0a704">python37Packages.skybellpy</a></li></li>
    <li><a href="https://gist.github.com/33d124f25b7ccf7c2e44b33c4f35bf01">python38Packages.skybellpy</a></li></li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `asynctest`.
Near pkgs/development/python-modules/skybellpy/default.nix:2:3:
```
  |
2 | , asynctest
  |   ^
```
</li>
    <li>warning: unused-argument

Unused argument: `pytest-mock`.
Near pkgs/development/python-modules/skybellpy/default.nix:7:3:
```
  |
7 | , pytest-mock
  |   ^
```
</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112991` at 6d78f158 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>python39Packages.skybellpy</li>
  </ul>
</details>
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/c99c5d9c5b10aff7de43612f30fc06b2">python37Packages.skybellpy</a></li></li>
    <li><a href="https://gist.github.com/89d036e13b485d724f770997e28eab00">python38Packages.skybellpy</a></li></li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `asynctest`.
Near pkgs/development/python-modules/skybellpy/default.nix:2:3:
```
  |
2 | , asynctest
  |   ^
```
</li>
    <li>warning: unused-argument

Unused argument: `pytest-mock`.
Near pkgs/development/python-modules/skybellpy/default.nix:7:3:
```
  |
7 | , pytest-mock
  |   ^
```
</li>
  </ul>
</details>

**Comment by fabaff** [comment]
Depends on https://github.com/NixOS/nixpkgs/pull/112854 for Python 3.9.

File: pkgs/development/python-modules/skybellpy/default.nix [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{ lib
+, asynctest
+, buildPythonPackage
+, colorlog
+, fetchFromGitHub
+, pytest-cov
+, pytest-mock
```
Comment (mweinelt) [inline pkgs/development/python-modules/skybellpy/default.nix:7]: unused

File: pkgs/development/python-modules/skybellpy/default.nix [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{ lib
+, asynctest
```
Comment (mweinelt) [inline pkgs/development/python-modules/skybellpy/default.nix:2]: unused

File: pkgs/development/python-modules/skybellpy/default.nix [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+, buildPythonPackage
+, colorlog
+, fetchFromGitHub
+, pytest-cov
```
Comment (mweinelt) [inline pkgs/development/python-modules/skybellpy/default.nix:6]: Please disable coverage if possible.

**Review by mweinelt — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/skybellpy/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/skybellpy/default.nix]: Thanks, removed.

File: pkgs/development/python-modules/skybellpy/default.nix (hunk 2 above)
Comment (fabaff) [reply pkgs/development/python-modules/skybellpy/default.nix]: Thanks, removed.

File: pkgs/development/python-modules/skybellpy/default.nix (hunk 3 above)
Comment (fabaff) [reply pkgs/development/python-modules/skybellpy/default.nix]: Done

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 112991` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>python37Packages.skybellpy</li>
    <li>python38Packages.skybellpy</li>
    <li>python39Packages.skybellpy</li>
  </ul>
</details>

**Pull request merged**